pkg/util/httproute/httproute.go

```diff
@@ -41,6 +42,26 @@ func (r Route) WithPathPattern(pathPattern string) Route {
 	return r
 }
 
+// TODO: It may not works as expected if the path's pathern is using `/` for "regex"(?)
```

Comment:
We do not support regex in Route. What do you mean?

Reply:
oh I thought `httproute.Route` support regex. but seems it only supports path param and `*`